spec.emu

```diff
@@ -432,270 +477,352 @@ contributors: Ujjwal Sharma, Younies Mahmoud
-        1. Perform ? RequireInternalSlot(_df_, [[InitializedDurationFormat]]).
-        1. Let _record_ be ? ToDurationRecord(_duration_).
-        1. Let _parts_ be PartitionDurationFormatPattern(_df_, _record_).
-        1. Let _result_ be a new empty String.
-        1. For each Record { [[Type]], [[Value]] } _part_ in _parts_, do
-          1. Set _result_ to the string-concatenation of _result_ and _part_.[[Value]].
-        1. Return _result_.
-      </emu-alg>
-    </emu-clause>
-    <emu-clause id="sec-Intl.DurationFormat.prototype.formatToParts">
-      <h1>Intl.DurationFormat.prototype.formatToParts ( _duration_ )</h1>
-
-      <p>When the `formatToParts` method is called with an argument _duration_, the following steps are taken:</p>
-
-      <emu-alg>
-        1. Let _df_ be *this* value.
-        1. Perform ? RequireInternalSlot(_df_, [[InitializedDurationFormat]]).
-        1. Let _record_ be ? ToDurationRecord(_duration_).
-        1. Let _parts_ be PartitionDurationFormatPattern(_df_, _record_).
-        1. Let _result_ be ! ArrayCreate(0).
-        1. Let _n_ be 0.
-        1. For each { [[Type]], [[Value]] } _part_ in _parts_, do
-          1. Let _obj_ be OrdinaryObjectCreate(%ObjectPrototype%).
-          1. Perform ! CreateDataPropertyOrThrow(_obj_, *"type"*, _part_.[[Type]]).
-          1. Perform ! CreateDataPropertyOrThrow(_obj_, *"value"*, _part_.[[Value]]).
-          1. If _part_.[[Unit]] is set, Perform ! CreateDataPropertyOrThrow(_obj_, *"unit"*, _part_.[[Unit]]).
-          1. Perform ! CreateDataPropertyOrThrow(_result_, ! ToString(_n_), _obj_).
-          1. Increment _n_ by 1.
-        1. Return _result_.
-      </emu-alg>
-    </emu-clause>
-    <emu-clause id="sec-Intl.DurationFormat.prototype.resolvedOptions">
-      <h1>Intl.DurationFormat.prototype.resolvedOptions ( )</h1>
-
-      <p>This function provides access to the locale and options computed during initialization of the object.</p>
-
-      <emu-alg>
-        1. Let _df_ be the *this* value.
-        1. Perform ? RequireInternalSlot(_df_, [[InitializedDurationFormat]]).
-        1. Let _options_ be OrdinaryObjectCreate(%ObjectPrototype%).
-        1. For each row of <emu-xref href="#table-durationformat-resolvedoptions-properties"></emu-xref>, except the header row, in table order, do
-          1. Let _p_ be the Property value of the current row.
-          1. Let _v_ be the value of _df_'s internal slot whose name is the Internal Slot value of the current row.
-          1. Assert: _v_ is not *undefined*.
-          1. Perform ! CreateDataPropertyOrThrow(_options_, _p_, _v_).
-        1. Return _options_.
-      </emu-alg>
-
-      <emu-table id="table-durationformat-resolvedoptions-properties">
-        <emu-caption>Resolved Options of DurationFormat Instances</emu-caption>
+      <emu-table id="table-durationformat">
+      <emu-caption>Selected internal slots and property names of DurationFormat instances</emu-caption>
```

Comment:
Why is it "Selected internal slots"? not just "Internal slots" ?

Reply:
Hm, yeah, that wording is ambiguous. The thing I was trying to highlight is that this is a subset of the internal slots, leaving out ones that weren't relevant to the function it's referenced in.

Reply:
so... which Internal Slots is not listed in here?